### ISSUE TITLE
Base text(s) for Greek

### ISSUE THREAD

**Issue by jonathanrobie** [biblicalhumanities/copenhagen-alliance#9]
We currently have projects using the following Greek texts:

- Tischendorf
- Nestle1904
- Bunning Heuristic Prototype
- SBLGNT
- NA 26/27/28
- Robinson-Pierpoint

These take very different approaches to morphology, and some take different approaches to syntax trees (e.g. PROIEL on Tischendorf).  The resources available for each text differ.

Tyndale House has come out with the Tyndale House Greek New Testament.  The Orthodox Church uses Antoniades 1904, a Byzantine text.

Can we agree on a set of critical texts to target and provide a full set of resources for?

**Comment by jonathanrobie** [comment]
For Greek, I think a full set of resources includes:


- base text
- lemmas
- morphology
- syntax trees (recursive)
- discourse analysis
- context-sensitive glosses
- lexicon
- reference grammar
- reference system to identify objects in each of these resources

**Comment by ernstboogert** [comment]
At the conference, we already discussed shortly the problem of _one_ basetext. One possible solution is to have an open licensed base text containing all parts of the Greek New Testament on which all modern editions of the Greek New Testament agree, together with a full collation of those editions for the remaining parts in a database including all morphology, syntax trees etc. 

This has several advantages:
1) the user can use his/her own edition (by choosing the appropriate variant readings collection)
2) morphology, syntax trees etc. align with the choice of the user
3) the user has not to look for a whole bunch of editions to play with but can simply use this flexible base text
4) new editions can easily be added to the database by incorporating their variant readings
5) editions are only present in the database as collections of variant readings, not as full texts, thus bypassing complex licensing issues (is this true?!)

I am very curious what the other CA-members think...!

**Comment by johanovic** [comment]
@ernstboogert Can you give an example of how you format that basetext with collations, for instance, in XML?

**Comment by jtauber** [comment]
It is certainly Ulrik and my intention to provide morphology and lemmatisation for a wide variety of texts. That's harder to do for some resources but I think morphology and lemmatisation both suit a "superset" approach.

**Comment by ernstboogert** [comment]
@johanovic No, not at the moment. It was simply an idea to think about... I think that it will be important to have at least the base text and the collections of variant readings in _separate files_. In that way, no file can be charged to be a copy of, for instance, the NA28 text. In the base text, places of variance can then be marked and filled with data from the other file, if displayed.

**Comment by jonathanrobie** [comment]
Alan Bunning has aligned texts for both manuscripts before AD 400 and critical texts. You can download them here:

https://greekcntr.org/downloads/CNTR-XML.zip

**Comment by jonathanrobie** [comment]
Aligned texts are very important, but many things are hard to do across all possible texts, e.g.

- context-sensitive glosses
- versification (which may differ among critical editions even when the text is the same)
- syntax trees (which are sensitive to exact wording)

In general, anything that pertains only to an individual word works pretty well across texts, anything that requires sensitivity to the sentence as a whole is dependent on a specific text.  Actually, you can get away with it a lot of the time, but the times you can't really stick out and cause problems.
